nemo/collections/asr/parts/asr_module_utils.py

```diff
@@ -0,0 +1,82 @@
+# Copyright (c) 2021, NVIDIA CORPORATION.  All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Optional
+
+from omegaconf import DictConfig, open_dict
+
+from nemo.collections.asr.modules import conv_asr
+from nemo.collections.asr.parts import jasper
+from nemo.utils import logging
+
+
+def change_conv_asr_se_context_window(model: 'ASRModel', context_window: int, update_config: bool = True):
```

Comment:
This method always need a model as the input, how about moving it into the ASRModuleMixin as user is supposed to call it via the model?